docs/make_local.jl

```diff
@@ -1,21 +1,25 @@
+module BuildDoc
 using Oscar
+using Kroki
```

Comment:
(Why) is this necessary?

Reply:
it is not - and this is still the main problem. I do not have Kroki installed (outside doc) thus this part fails. It will work if I do the activate bit, but them I am not sure I fully understand which packages at what version are actually used. It appears that if I `activate` after a `using`, the used packages remain, but any new package is then loaded from the new Project.toml. Not sure what the resolve process will make of this.
I was hoping (in vain) that I can trick julia into accpeting Kroki by e.g. me doing `using Kroki` before in some way...